backend/hitas/views/apartment.py

```diff
@@ -34,6 +37,55 @@ class Meta:
         ]
 
 
+class DepreciationPercentageField(serializers.ChoiceField):
+    def __init__(self, **kwargs):
+        super().__init__(choices=DepreciationPercentage.choices(), **kwargs)
+
+    def to_representation(self, percentage: DepreciationPercentage):
+        match percentage:
+            case DepreciationPercentage.TEN:
+                return 10.0
+            case DepreciationPercentage.TWO_AND_HALF:
+                return 2.5
+            case DepreciationPercentage.ZERO:
+                return 0.0
+            case _:
+                raise NotImplementedError(f"Value '{percentage} not implemented.")
```

Comment:
Missing the closing `'` after `'{percentage}` :D

Reply:
fixed :)